src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Circle extends Figure {
+    private double rad;
+    private double area;
+    private String color = new ColorSupplier().getRandomColor();
```

Comment:
1. You don't need to store the value of `area` - calculate and return it with proper method.
2. No need to set a random color here - you don't set a random values for `rad` - make these values be provided as constructor arguments.
3. Why do you store the color as string when you have dedicated enum type?